packages/clerk-js/rspack.config.js

```diff
@@ -320,10 +342,13 @@ const prodConfig = ({ mode }) => {
       // Include the lazy chunks in the bundle as well
       // so that the final bundle can be imported and bundled again
       // by a different bundler, eg the webpack instance used by react-scripts
-      new webpack.optimize.LimitChunkCountPlugin({
+      new rspack.optimize.LimitChunkCountPlugin({
         maxChunks: 1,
       }),
     ],
+    optimization: {
```

Comment:
This was added because the `LimitChunkCountPlugin` doesn't reliably result in a single file.